src/wizard/wizard.scss

```diff
@@ -2,168 +2,147 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
-@mixin iui-wizards {
+@mixin iui-wizard {
   @include iui-reset;
   @include iui-font-family;
 
-  > .iui-wizards-wrapper {
+  > .iui-wizard-container {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
     align-items: flex-start;
     width: 100%;
     margin-bottom: $iui-baseline * 0.5;
 
-    > .iui-wizards-step {
+    .iui-wizard-step {
+      flex: 1 1 0;
       align-items: center;
       display: flex;
-      flex-direction: column-reverse;
+      flex-direction: column;
       font-size: $iui-font-size-small;
       font-weight: $iui-font-weight-normal;
       line-height: $iui-line-height;
       position: relative;
-      width: 20%;
-
       @include themed {
         color: t(iui-color-background-positive);
       }
 
-      &:hover {
-        text-decoration: none;
-      }
+      .iui-step-track {
+        display: flex;
+        align-items: center;
+        width: 100%;
 
-      > .iui-wizards-step-title {
-        text-align: center;
-        @include themed {
-          color: t(iui-text-color-muted);
+        .iui-track {
+          height: $iui-xxs;
+          flex: 1 2 auto;
+          @include themed {
+            background-color: t(iui-color-background-5);
+          }
         }
-      }
 
-      > .iui-wizards-step-circle {
-        align-items: center;
-        border-radius: 50%;
-        display: flex;
-        height: $iui-l;
-        width: $iui-l;
-        justify-content: center;
-        overflow-wrap: break-word;
-        padding: 0;
-        position: relative;
-        text-align: center;
-        z-index: 1;
-        user-select: none;
-        @include themed {
-          border: 1px solid t(iui-color-background-5);
-          background-color: t(iui-color-background-1);
-          color: t(iui-text-color-muted);
+        .iui-circle {
+          align-items: center;
+          border-radius: 50%;
+          display: flex;
```

Comment:
Can you check whether all these flex rules are needed?

Reply:
I was able to remove the flex rules and instead use line-height to center the text within the circle.

Reply:
line-height is worse tho

Reply:
Why is line-height worse?

Reply:
It's less predictable and will randomly stop working if you change something.

Reply:
Reverted back to flex, but was able to keep `flex-shrink: 0` removed.